packages/internal/src/build/api.ts

```diff
@@ -35,18 +35,18 @@ export const prebuildApiFiles = (srcFiles: string[]) => {
   const plugins = getBabelPlugins()
 
   return srcFiles.map((srcPath) => {
-    const result = prebuildFile(srcPath, plugins)
+    let dstPath = path.relative(rwjsPaths.base, srcPath)
```

Comment:
I moved the declaration of `dstPath` up because we are now passing `dstPath` as parameter to the `prebuildFile` function. This is necessary so the `prebuildFile` function can resolve the relative path from `dstPath` to `srcPath`, and capture that in the source map (as a relative path). This relative path will later be discovered by ESBuild